engine/src/main/java/com/datatorrent/stram/StramLocalCluster.java

```diff
@@ -318,6 +329,18 @@ public StramLocalCluster(LogicalPlan dag) throws IOException, ClassNotFoundExcep
     }
   }
 
+  private void addToClasspath(String jar)
```

Comment:
@vrozov Moved that common code that should be done only once to caller of this method.